docs/topics/Permissions.md

```diff
@@ -48,6 +48,14 @@ Permissions follow a hierarchy with the following roles:
 * Bots can only sort roles lower than their highest role.
 * Bots can only kick/ban users of with a lower highest role than themselves.
 
+In the case of permissions overlapping with each other—e.g. a user's role is banned from seeing a channel, but the user has a specific override—they are applied to a user in the following order:
+
+1. Base permissions (@everyone) are applied
+2. All permissions for the roles assigned to a user are applied
```

Comment:
might be useful to clarify how these guild-level permissions are applied (roles are all ORed, hierarchy only matters for cases dictated above), but then again maybe that's obvious